<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelRepositoryForm</class>
 <widget class="QWidget" name="ModelRepositoryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Repository</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="leftWidget" native="true">
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QLineEdit" name="searchBox">
         <property name="placeholderText">
          <string>Search models...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="modelTree">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rightWidget" native="true">
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <widget class="QToolBar" name="toolBar">
         <addaction name="actionNew"/>
         <addaction name="actionOpen"/>
         <addaction name="actionSave"/>
         <addaction name="separator"/>
         <addaction name="actionCheckout"/>
         <addaction name="actionCheckin"/>
         <addaction name="separator"/>
         <addaction name="actionVersion"/>
         <addaction name="actionCompare"/>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="contentTabs">
         <widget class="QWidget" name="previewTab">
          <attribute name="title">
           <string>Preview</string>
          </attribute>
          <layout class="QVBoxLayout" name="previewLayout">
           <item>
            <widget class="QOpenGLWidget" name="modelViewer"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="propertiesTab">
          <attribute name="title">
           <string>Properties</string>
          </attribute>
          <layout class="QFormLayout" name="propertiesLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="nameEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="typeLabel">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="typeCombo"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="statusLabel">
             <property name="text">
              <string>Status:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="statusCombo"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="versionLabel">
             <property name="text">
              <string>Version:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="versionEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="descriptionLabel">
             <property name="text">
              <string>Description:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QTextEdit" name="descriptionEdit"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="historyTab">
          <attribute name="title">
           <string>History</string>
          </attribute>
          <layout class="QVBoxLayout" name="historyLayout">
           <item>
            <widget class="QTableView" name="historyTable"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Create new model</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open model</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save model</string>
   </property>
  </action>
  <action name="actionCheckout">
   <property name="text">
    <string>Checkout</string>
   </property>
   <property name="toolTip">
    <string>Checkout model</string>
   </property>
  </action>
  <action name="actionCheckin">
   <property name="text">
    <string>Checkin</string>
   </property>
   <property name="toolTip">
    <string>Checkin model</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
   <property name="toolTip">
    <string>Create new version</string>
   </property>
  </action>
  <action name="actionCompare">
   <property name="text">
    <string>Compare</string>
   </property>
   <property name="toolTip">
    <string>Compare versions</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
